src/frontend/efiling-demo/Dockerfile

```diff
@@ -7,7 +7,8 @@ WORKDIR /app
 
 COPY ./package.json ./yarn.lock ./
 
-RUN yarn install --production=true
+RUN yarn install --production=true \
+    && yarn add --dev react-app-rewired
```

Comment:
If we need react-app-rewired for production builds, should we add it to the dependencies in the package.json, instead of adding it to the dev dependencies after a production install?